.travis.yml

```diff
@@ -17,95 +17,102 @@ install:
   - hibench=$(pwd)
   - echo ${hibench}
   - cd /opt/
-  - wget https://archive.apache.org/dist/spark/spark-3.0.0/spark-3.0.0-bin-hadoop2.7.tgz
-  - tar -xzf spark-3.0.0-bin-hadoop2.7.tgz
   - wget https://archive.apache.org/dist/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz
   - tar -xzf spark-2.4.0-bin-hadoop2.7.tgz
+  - wget https://archive.apache.org/dist/spark/spark-3.0.0/spark-3.0.0-bin-hadoop3.2.tgz
+  - tar -xzf spark-3.0.0-bin-hadoop3.2.tgz
   - wget https://archive.apache.org/dist/hadoop/core/hadoop-2.7.7/hadoop-2.7.7.tar.gz
   - tar -xzf hadoop-2.7.7.tar.gz
+  - wget https://archive.apache.org/dist/hadoop/core/hadoop-3.2.1/hadoop-3.2.1.tar.gz
+  - tar -xzf hadoop-3.2.1.tar.gz
   - cd ${hibench}
-  - cp ./travis/spark-env.sh /opt/spark-3.0.0-bin-hadoop2.7/conf/
   - cp ./travis/spark-env.sh /opt/spark-2.4.0-bin-hadoop2.7/conf/
+  - cp ./travis/spark-env.sh /opt/spark-3.0.0-bin-hadoop3.2/conf/
   - cp ./travis/core-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/hdfs-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/mapred-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/yarn-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/hadoop-env.sh /opt/hadoop-2.7.7/etc/hadoop/
-  - cp ./travis/hibench.conf ./conf/
-  - cp ./travis/hadoop.conf ./conf/
+  - cp ./travis/core-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/hdfs-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/mapred-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/yarn-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/hadoop-env.sh /opt/hadoop-3.2.1/etc/hadoop/
 jobs:
   include:
     - name: build-multi-version
       script:
-        - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.4 -Dscala=2.11
+        - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.4 -Dscala=2.11 -Dhadoop=2.7 -Dhive=0.14
+        - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.4 -Dscala=2.11 -Dhadoop=3.0 -Dhive=3.0
+        - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.4 -Dscala=2.11 -Dhadoop=3.1 -Dhive=3.0
         - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=3.0 -Dscala=2.12
 
-    - name: spark-3.0_micro
+    - name: spark_2.4_hadoop-2.7_micro
       before_script:
-        - ./travis/export_environment_variables_spark_3.0.sh
+        - source ./travis/export_env_spark_2.4_hadoop_2.7.sh
```

Comment:
Use dot?

Reply:
@gczsjdy  Using dot can't work well. So I used source instead of dot.  

![image](https://user-images.githubusercontent.com/45281494/89747234-5f0e4b80-daf0-11ea-972c-18abba97af61.png)

Reply:
@gczsjdy  it's done. Using dot instead of source.